test/functional/apps/management/_index_pattern_create_delete.js

```diff
@@ -39,17 +35,16 @@ export default function ({ getService, getPageObjects }) {
       });
     });
 
-    // FLAKY: https://github.com/elastic/kibana/issues/107831
-    describe.skip('validation', function () {
+    describe('validation', function () {
       it('can display errors', async function () {
         await PageObjects.settings.clickAddNewIndexPatternButton();
-        await PageObjects.settings.setIndexPatternField('log*');
+        await PageObjects.settings.setIndexPatternField('log-fake*');
         await (await PageObjects.settings.getSaveIndexPatternButton()).click();
         await find.byClassName('euiFormErrorText');
       });
 
       it('can resolve errors and submit', async function () {
-        await PageObjects.settings.selectTimeFieldOption('@timestamp');
```

Comment:
Changing test logic a bit here because now we auto-select `@timestamp` if available